include/kodlab_mjbots_sdk/robot_interface.h

```diff
@@ -0,0 +1,143 @@
+/*!
+ * @file robot_interface.h
+ * @author Kodlab - J. Diego Caporale (jdcap@seas.upenn.edu)
+ * @brief Robot Interface Class 
+ * @date 2022-07-12
+ * 
+ * @copyright Copyright (c) 2021 The Trustees of the University of Pennsylvania. All Rights Reserved
+ *            BSD 3-Clause License
+ * 
+ */
+#pragma once 
+
+#include <vector>
+#include <memory>
+#include "kodlab_mjbots_sdk/common_header.h"
+#include "kodlab_mjbots_sdk/joint_base.h"
+#include "kodlab_mjbots_sdk/soft_start.h"
+
+namespace kodlab
+{
+    /*!
+     * @brief Sets up a kodlab robot interface class 
+     * 
+     */
+    class RobotInterface {
+    public:
+        static const int KILL_ROBOT = -1;
```

Comment:
```suggestion
        static const int KILL_ROBOT = -1;  // Kill mode/behavior index, which will be used to signal E-Stopping the robot
```

Reply:
Resolved in 571f61791965a974d1ae6976b4ec54d24ca269b0.